endpoints/auth.go

```diff
@@ -185,7 +184,7 @@ func getCertExpirationTime(h http.Header) time.Duration {
 
 // getCachedCerts fetches public certificates info from DefaultCertURI and
 // caches it for the duration specified in Age header of a response.
-func getCachedCerts(c Context) (*certsList, error) {
+func getCachedCerts(c context.Context) (*certsList, error) {
```

Comment:
Since we're changing so much code: rename this cachedCerts, it's more idiomatic.